cohortextractor/patients.py

```diff
@@ -667,19 +878,37 @@ def admitted_to_hospital(
     with_these_procedures=None,
     return_expectations=None,
 ):
-    """Return information about admission to hospital.
-
-    Options for `returning` are:
-
-        binary_flag: Whether patient was admitted to hospital
-        primary_diagnosis: ICD-10 code of primary diagnosis
-        date_admitted: Date patient was admitted to hospital
-        date_discharged: Date patient was discharged from hospital
-
-    `with_these_diagnoses` is optional, and is a list of ICD-10 codes
-    `with_these_primary_diagnoses` is optional, and is a list of ICD-10 codes
-    `with_these_procedures` is optional, and is a list of OPCS-4 codes
-
-    See https://github.com/opensafely/cohort-extractor/issues/186 for discussion.
+    """"
+    Return information about admission to hospital.
+
+    See https://github.com/opensafely/cohort-extractor/issues/186 for in-depth discussion and background.
+
+    Args:
+        on_or_before (str): The latest date of admission
+        on_or_after (str): The earliest date of admission
+        between (list): A tuple of [`on_or_after`, `on_or_before`]
+        returning (str): One of `binary_flag` (if they were admitted at all), `date_admitted`, `date_discharged`, `number_of_matches_in_period`, `primary_diagnosis`
+        find_first_match_in_period (bool): For date return values, always choose the earliest matching occurence
```

Comment:
does this have a (meaningful) default thath should be noted? on the face of it, looks like default is `None` which is (iirc) falsey?

Reply:
ah, seems like it's picking up the default automatically (this is the function from your example screenshot?)